io.github.streetpea.chiaki4deck.yaml

```diff
@@ -0,0 +1,130 @@
+rename-desktop-file: chiaki4deck.desktop
+separate-locales: false
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --allow=bluetooth
+  - --socket=pulseaudio
+  - --own-name=org.kde.*
+  - --env=DBUS_FATAL_WARNINGS=0
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --filesystem=xdg-run/pipewire-0
+  
+modules:
+  - name: protobuf-compilers
+    sources:
+      - type: archive
+        url: https://github.com/protocolbuffers/protobuf/releases/download/v21.12/protobuf-cpp-3.21.12.tar.gz
+        sha256: 4eab9b524aa5913c6fffb20b2a8abf5ef7f95a80bc0701f3a6dbb4c607f73460
+
+  - name: ffmpeg
+    cleanup:
+      - /bin/ffmpeg
+      - /lib/*.so
+      - /include
+      - /lib/pkgconfig
+      - /share/ffmpeg/examples
+      - /share/ffmpeg
+    config-opts:
+      - --disable-all
+      - --enable-avcodec
+      - --enable-decoder=h264
+      - --enable-decoder=hevc
+      - --enable-hwaccel=h264_vaapi
+      - --enable-hwaccel=hevc_vaapi
+    sources:
+      - type: archive
+        url: https://ffmpeg.org/releases/ffmpeg-6.0.tar.xz
+        sha256: 57be87c22d9b49c112b6d24bc67d42508660e6b718b3db89c44e47e289137082
+
+  - name: python3-google
+    buildsystem: simple
+    build-commands:
+      - "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} google"
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/3e/db/5ba900920642414333bdc3cb397075381d63eafc7e75c2373bbc560a9fa1/soupsieve-2.0.1.tar.gz
+        sha256: a59dc181727e95d25f781f0eb4fd1825ff45590ec8ff49eadfd7f1a537cc0232
+      - type: file
+        url: https://files.pythonhosted.org/packages/e8/b0/cd2b968000577ec5ce6c741a54d846dfa402372369b8b6861720aa9ecea7/beautifulsoup4-4.11.1.tar.gz
+        sha256: ad9aa55b65ef2808eb405f46cf74df7fcb7044d5cbc26487f96eb2ef2e436693
+      - type: file
+        url: https://files.pythonhosted.org/packages/89/97/b49c69893cddea912c7a660a4b6102c6b02cd268f8c7162dd70b7c16f753/google-3.0.0.tar.gz
+        sha256: 143530122ee5130509ad5e989f0512f7cb218b2d4eddbafbad40fd10e8d8ccbe
+
+  - name: python3-protobuf
+    buildsystem: simple
+    build-commands:
+      - "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} protobuf"
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/ba/dd/f8a01b146bf45ac12a829bbc599e6590aa6a6849ace7d28c42d77041d6ab/protobuf-4.21.12.tar.gz
+        sha256: 7cd532c4566d0e6feafecc1059d04c7915aec8e182d1cf7adee8b24ef1e2e6ab
+
+  - name: SDL2
+    config-opts:
+      - --disable-arts
+      - --disable-esd
+      - --disable-nas
+      - --disable-alsa
+      - --disable-oss
+      - --disable-sndio
+      - --enable-video-wayland
+      - --enable-wayland-shared=no
+      - --enable-video-kmsdrm
+      - --disable-rpath
+    sources:
+      - type: git
+        url: https://github.com/libsdl-org/SDL.git
+        tag: release-2.28.1
```

Comment:
the SDK has 2.24. isn't it enough?

Reply:
also it should cleanup development files like includs , static lib and other.

Reply:
the DualSense edge mappings were added in 2.26.5 and some people want to use that controller for PlayStation remote play